recipes/scitime/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.0.1" %}
+
+package:
+  name: scitime
+  version: {{ version }}
+
+source:
+  url: https://github.com/nathan-toubiana/scitime/archive/{{ version }}.tar.gz
+  sha256: 05f11512343bbaf521c6358e8321a8555d0ef4fe41a76ced9a8234d8b8bbab5b
+
+build:
+  number: 1
```

Comment:
This should be zero in staged recipes.

Reply:
thanks for the review! should be alright now